.release/buildspec_integ.yml

```
@@ -0,0 +1,65 @@
+version: 0.2
+env:
+  parameter-store:
+    NEUTRAL_BUILD_RELEASE_ROLE: COPILOT_NEUTRAL_BUILD_RELEASE_ROLE_ARN
+    NEUTRAL_BUILD_RELEASE_BUCKET_NAME: COPILOT_NEUTRAL_BUILD_RELEASE_BUCKET_NAME
+  variables: 
+    AWS_STS_REGIONAL_ENDPOINTS: regional
+    INTEG_TEST_SESSION_NAME: aws_copilot_cli_integ_test
+    TAG_KEY: access
+    TAG_VALUE: private
+
+phases:
+  install:
+    runtime-versions:
+      golang: 1.13
+      nodejs: 10
+  build:
+    commands:
+      - echo `git rev-parse HEAD` # Do not delete; for pipeline logging purposes
+      - cd $CODEBUILD_SRC_DIR
+      - export GOPATH=/go
+      - rm -rf cf-custom-resources/node_modules
+      - |
+        tag_artifacts() {{
+            CREDS=`aws sts assume-role --region $AWS_DEFAULT_REGION --role-arn "$NEUTRAL_BUILD_RELEASE_ROLE" --role-session-name $INTEG_TEST_SESSION_NAME`
```

Let's up the duration of the sessions to 2 hours. 
```suggestion
            CREDS=`aws sts assume-role --duration-seconds 7200 --role-arn "$NEUTRAL_BUILD_RELEASE_ROLE" --role-session-name $INTEG_TEST_SESSION_NAME`
```